docs/user-guide/api-mediation/api-gateway-configuration.md

```diff
@@ -103,6 +105,8 @@ where:
 
 Update the user password using the z/OSMF Authentication provider. To use this functionality, add the parameter `newPassword` on the login endpoint `/gateway/api/v1/auth/login`. The Gateway service returns a valid JWT with the response code `204` as a result of successful password change. The user is then authenticated and can consume APIs through the Gateway. If it is not possible to change the password for any reason, the response code is `401`.
 
+This feature is also available in the API Catalog.
```

Comment:
I'm just wondering if this feature should be documented in the API Catalog doc as well... Is the process the same via the Catalog route?

Reply:
This functionality is present also in the catalog, but only through GW route